mysql-test/t/percona_bug_ps4988.test

```diff
@@ -0,0 +1,5 @@
+# PS-4988: Uninitailized btr stat mutex caused assertion failure
```

Comment:
Consider naming the test "percona_sync_debug", so that it can receive further similar tests in the future